### ISSUE TITLE
Unknown column 'Taxonomy.required'

### ISSUE THREAD

**Issue by Tomasuh** [MISP/MISP#4560]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug, support
| OS version (server)       | CentOS,
| OS version (client)       | 7.6
| PHP version               | 7.1
| MISP version / git hash   | v2.4.106
| Browser                   | If applicable

After upgrading the MISP to the latest version I get this error:
`2019-05-02 09:50:01 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Taxonomy.required' in 'where clause'
Request URL: /events/view/41278
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(502): PDOStatement->execute(Array)
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(468): DboSource->_execute('SELECT `Taxonom...', Array)
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(715): DboSource->execute('SELECT `Taxonom...', Array, Array)
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1226): DboSource->fetchAll('SELECT `Taxonom...', Array)
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3040): DboSource->read(Object(Taxonomy), Array)
#5 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3012): Model->_readDataSource('list', Array)
#6 /var/www/MISP/app/Model/Event.php(6236): Model->find('list', Array)
#7 /var/www/MISP/app/Controller/EventsController.php(1472): Event->getRequiredTaxonomies()
#8 /var/www/MISP/app/Controller/EventsController.php(1575): EventsController->__viewUI(Array, false, NULL)
#9 [internal function]: EventsController->view('41278')
#10 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(499): ReflectionMethod->invokeArgs(Object(EventsController), Array)
#11 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#12 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(EventsController), Object(CakeRequest))
#13 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}`

I followed the update instructions in https://misp.github.io/MISP/UPDATE/ and the update was minor, from 2.4.xx (don't remember exact version but belive it was around .90) to v2.4.106.

Any advice what might have caused this?
I tried to remove the cache files under /var/www/MISP/app/tmp/cache/.

**Comment by Tomasuh** [comment]
![table](https://user-images.githubusercontent.com/3432107/57065450-c3c1e000-6cc9-11e9-8c66-8b2c7e58e3d6.PNG)

Maybe completely unrelated :)

**Comment by mokaddem** [comment]
Hello @Tomasuh,
It seems that after the PULL, your database didn't get updated.
Could you check in your logs if anything went wrong?
You can find them in the web interface at:
``/admin/logs/index/filter:update``
or in your file system
``MISP/app/tmp/logs``

**Comment by Tomasuh** [comment]
Hello @mokaddem, thank you for looking into this!

Actually I was doing the manual update @ instructions at https://misp.github.io/MISP/UPDATE/ as I had some issues with the update through the admin panel (I discovered after that it was due to a typo in /etc/opt/rh/rh-php71/php-fpm.d/www.conf regarding the PATH variable, making git executable not to be found by MISP).

After your pointer that the database didn't get updated and I realized that it was done through code triggered through the administration panel update button I reset my repo to the commit I was at before the update.
And then ran the update through the administration panel, now it works!
